runtime/v2/builder.go

```diff
@@ -175,6 +175,19 @@ func (a *AppBuilder[T]) Build(opts ...AppBuilderOption[T]) (*App[T], error) {
 			}
 			return nil
 		},
+		ExportGenesis: func(ctx context.Context, version uint64) ([]byte, error) {
+			genesisJson, err := a.app.moduleManager.ExportGenesisForModules(ctx) // TODO: add option to export certain modules?
```

Comment:
```suggestion
			genesisJson, err := a.app.moduleManager.ExportGenesisForModules(ctx) 
```
dont think this feature is needed